openwisp_controller/config/migrations/0056_ow_zt-to-global.py

```diff
@@ -0,0 +1,18 @@
+from django.db import migrations
+import json
+
+def change_owzt_to_global(apps, schema_editor):
+    Template = apps.get_model('config', 'Template')
+    for template in Template.objects.all():
+        config = json.loads(template.config)
+        if 'zerotier' in config:
+            for item in config.get('zerotier',[]):
+                if item.get('name') == 'ow_zt':
+                    item['name'] = 'global'
+
+class Migration(migrations.Migration):
+    dependencies = [('config', '0055_alter_config_status')]
+
+    operations = [
+        migrations.RunPython(change_owzt_to_global, reverse_code=migrations.RunPython.noop)
+    ]
```

Comment:
configure your editor to always add a blank line at the end of files

Reply:
thank you for reviewing